apps/authentication-portal/src/main/webapp/includes/language-switcher.jsp

```diff
@@ -0,0 +1,146 @@
+<%--
+  ~ Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org). All Rights Reserved.
+  ~
+  ~  WSO2 Inc. licenses this file to you under the Apache License,
+  ~  Version 2.0 (the "License"); you may not use this file except
+  ~  in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+--%>
+
+<%@ page import="org.wso2.carbon.identity.application.authentication.endpoint.util.AuthenticationEndpointUtil" %>
+<%-- Localization --%>
+<jsp:directive.include file="localize.jsp" />
+
+<script src="libs/jquery_3.6.0/jquery-3.6.0.min.js"></script>
```

Comment:
Won't this be loaded from somewhere else as well?

Reply:
In some pages, It won't. That's why I added here as well